app/assets/javascripts/modules/moj.RevealAdditional.js

```diff
@@ -1,39 +1,74 @@
-(function () {
+(function() {
   'use strict';
 
   moj.Modules.RevealAdditional = {
     el: '.js-RevealAdditional',
 
-    init: function () {
+    init: function() {
       this.cacheEls();
       this.bindEvents();
     },
 
-    bindEvents: function () {
-      this.$actuators.on('change', this.actuate);
+    bindEvents: function() {
+      var self = this;
+      this.$select.on('change', function() {
+        self.actuate(this);
+      });
+      this.$addButton.on('click', function() {
+        self.addVisitor();
+      });
+      this.$removeButton.on('click', function() {
+        self.removeVisitor();
+      });
       moj.Events.on('render', $.proxy(this.render, this));
     },
 
-    cacheEls: function () {
-      this.$actuators = $(this.el);
+    cacheEls: function() {
+      this.$select = $(this.el).find('.js-RevealAdditionalSelect');
+      this.$addButton = $('.js-RevealAdditionalButton');
+      this.$removeButton = $('.js-HideAdditionalButton');
```

Comment:
This two elements aren't within the component ``` this.$el``` it seems that there should be a meta component wrapping these three elements and orchestrating the triggering of events. 
As a rule of thumbs each js component ```this.$el``` never reaches out of it's inner dom.